.github/workflows/ci.yaml

```diff
@@ -106,6 +106,33 @@ jobs:
           files: "manifests"
           version: ${{ matrix.k8s }}
 
+  template-test:
+    name: template-test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v3
+        with:
+          version: v3.6.3
+
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: stable
+
+      - name: golangci-lint
```

Comment:
This isn't used at the moment so we shouldn't install it.

Reply:
@MichelHollands Thank you! Just to clarify, we were not using it before, as it's the first time we introduce Go code to this repo.

The idea is to enforce standard rules from day one, so that we don't need to fix a lot of lint errors when we later introduce it!
This action (with no args, you see) runs the standard linters as far as I know.

Reply:
Sorry, I misread this as just installing it.

Reply:
No problem at all! The same happened to me for the first time I saw it 😂